devices/sinope.js

```diff
@@ -1179,21 +1191,26 @@ module.exports = [
         model: 'RM3500ZB',
         vendor: 'Sinopé',
         description: 'Calypso smart water heater controller',
-        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering, fz.ias_water_leak_alarm_1, fz.temperature],
+        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering, fzLocal.ias_water_leak_alarm,
+            fzLocal.sinope, fz.temperature],
         toZigbee: [tz.on_off],
         exposes: [e.switch(), e.power(), e.current(), e.voltage(), e.energy(), e.water_leak(), e.temperature()],
         configure: async (device, coordinatorEndpoint) => {
             const endpoint = device.getEndpoint(1);
-            const binds = ['genOnOff', 'haElectricalMeasurement', 'seMetering', 'Temperature'];
+            const binds = ['genOnOff', 'haElectricalMeasurement', 'seMetering', 'msTemperatureMeasurement', 'ssIasZone',
+                'manuSpecificSinope'];
             await reporting.bind(endpoint, coordinatorEndpoint, binds);
             await reporting.onOff(endpoint);
+            await reporting.temperature(endpoint, {min: 10, max: 301, change: 10}); // divider 100: 0.1C
             await reporting.readEletricalMeasurementMultiplierDivisors(endpoint);
-            await reporting.activePower(endpoint);
-            await reporting.rmsCurrent(endpoint);
-            await reporting.rmsVoltage(endpoint);
+            await reporting.activePower(endpoint, {min: 10, max: 305, change: 2}); // divider 1 : 2W
+            await reporting.rmsCurrent(endpoint, {min: 10, max: 306, change: 10}); // divider 1000: 0.01Arms
+            await reporting.rmsVoltage(endpoint, {min: 10, max: 307, change: 1}); // divider 1: 1Vrms
```

Comment:
Why 305/306 and 307?

Reply:
In the past, we observed a lot of traffic and packet drops caused by devices reporting multiple electric measurements updates. This happens when there is a large number of devices on the same Zigbee network. In my case I have over 15 thermostats of all kind and other devices. Event report packet storms are amplified by power restoration or voltage variations. The packet storm will repeat afterward at every maximum reporting interval.  
Also Sinope thermostats and other of their devices doing energy or electrical measurements are verry chatty and sensitive to voltage variations. I experienced this with their thermostats and applied different maximum reporting intervals to the TH1123ZB/TH1124ZB/TH1500ZB converters in previous PR. , Their new water heater controler (RM3500ZB) and their wall plug switch (SP2600ZB) are even more sensitive.
Using different maximum reporting intervals is a proven method also used by Smarthings converters and others to mitigate this problem.